### PR TITLE
docs: reorg the structure of slashing module

### PR DESCRIPTION
**Pull request by guagualvcha**
### Description
Minor change about the slash module.

### Example
No

### Changes
1. Modify the title from `Slashing and Jail` to `Overview`
2. Add more content on the `overview` page.
3. Modify the title from `Monitoring and Querying Slashing Events` to `Monitoring`
4. Combine the `cenarios-slash.md` into `overview`

### CODE REVIEW

File: docs/validator/Penalty-overview.md [hunk 1]
```diff
@@ -1,14 +1,26 @@
 ---
-sidebar_label: Slashing and Jail
+sidebar_label: Overview
 hide_table_of_contents: false
 sidebar_position: 2
 ---
 
-# Penalty for Bad Behaviour by Validators
+# Overview
+Slashing is part of the on-chain governance, to ensure the malicious or negative behaviors are punished. BSC slash can be submitted by anyone. The transaction submission requires slash evidence and cost fees but also brings a larger reward when it is successful.
```
Comment (forcodedancing) [inline docs/validator/Penalty-overview.md:8]: `cost` -> `costs`

File: docs/validator/Penalty-overview.md [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+Anyone can submit a slash request on BC with the evidence of Double Sign of BSC, which should contain the 2 block headers with the same height and parent block, sealed by the offending validator. Upon receiving the evidence, if the BC verifies it to be valid:
+
+The validator will be removed from validator set by an instance BSC validator set update Cross-Chain update;
+A predefined amount of BNB would be slashed from the self-delegated BNB of the validator; Both validator and its delegators will not receive the staking rewards.
+Part of the slashed BNB will allocate to the submitter’s address, which is a reward and larger than the cost of submitting slash request transaction
```
Comment (forcodedancing) [inline docs/validator/Penalty-overview.md:19]: `allocate` -> `be allocated`

File: docs/validator/Penalty-overview.md [hunk 3]
```diff
@@ -1,14 +1,26 @@
...
+The rest of the slashed BNB will allocate to the other validators’ custody addresses, and distributed to all delegators in the same way as blocking reward.
```
Comment (forcodedancing) [inline docs/validator/Penalty-overview.md:20]: `allocate` -> `be allocated`

File: docs/validator/Penalty-overview.md (hunk 1 above)
Comment (guagualvcha) [reply docs/validator/Penalty-overview.md:8]: fixed

File: docs/validator/Penalty-overview.md (hunk 2 above)
Comment (guagualvcha) [reply docs/validator/Penalty-overview.md:19]: fixed

**Pull request merged**